tests/fixed/example-class.php

```diff
@@ -129,6 +129,9 @@ public function trySwitchSpace() : void
         try {
             $var = 1;
             switch (self::VERSION) {
+                case 1:
+                case 2:
+                    // do something
```

Comment:
Could you add a third `case` with a different action? Just to make sure that we're testing the expected :)

Reply:
Added, good?